cards.js

```diff
@@ -16,8 +16,46 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
+function buildDeck(values, suits){
+  var deck = [];
+      for( var suitIndex = 0; suitIndex < suits.length; suitIndex++ ) {
+      for( var valueIndex = 0; valueIndex < values.length; valueIndex++ ) {
+        cardValue = values[valueIndex];
+        cardSuit = suits[suitIndex];
+        newCard =  {
+          cardSuit: cardSuit,
+          cardValue: cardValue,
+        // n+1
+        }
+        deck.push(newCard)
+      }
+   }
+  return deck;
+}
+
+var deck = buildDeck(values, suits);
+console.log(deck);
+console.log("*******************************")
+
+var shuffledDeck = shuffle(deck);
+console.log(shuffledDeck);
+console.log("*******************************")
 
-// Put some code in here!
+var player1Card = shuffledDeck.pop()
+console.log(player1Card)
+console.log("*******************************")
 
+//console.log(shuffledDeck.slice(26))
+//console.log(shuffledDeck.pop());
+
+var player2Card = shuffledDeck.pop()
+console.log(player2Card)
+console.log("*******************************")
+
```

Comment:
everything looks great up to here!

All you need to do is find a way to compare the two cards, which you came close to doing. Maybe take a look and see how other people did it?

(Note that you never call your `highCard` function here, so it's not really doing anything!